interfaces/AsigurareInterface.java

```diff
@@ -0,0 +1,14 @@
+package interfaces;
+
+import entities.Asigurare;
+
+public interface AsigurareInterface {
```

Comment:
[Fix] nu e nevoie sa o denumesti Interface, te ajuta cu asta IDE-ul, ca apare cu alta iconita in pachet